spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -491,13 +491,7 @@ public void declareQueue(String beanName, Queue queue) {
 			}
 		}
 		catch (UncategorizedAmqpException e) {
```

Comment:
Since the sole purpose of the removed check is the test we're removing, we can remove the try-catch as well